public/app/plugins/panel/geomap/style/types.ts

```diff
@@ -30,6 +30,10 @@ export interface StyleConfig {
   // Can show markers and text together!
   text?: TextDimensionConfig;
   textConfig?: TextStyleConfig;
+
+  // Allow for rotation of markers
+  isDegree?: boolean;
+  rotation?: ScaleDimensionConfig;
```

Comment:
rather than `ScaleDimensionConfig` should we add `ScalerDimensionConfig`?

ScaleDimensionConfig will convert the value to something between 0-1, but I think we want to be able to use the raw float value directly.

Lets remove the 'isDegree' option, and assume the value from rotation is degrees -- we can add transforms later if necessary

Reply:
Scalar